server/model/maintenance.js

```diff
@@ -395,10 +411,7 @@ class Maintenance extends BeanModel {
         } else if (!this.strategy.startsWith("recurring-")) {
             this.cron = "";
         } else if (this.strategy === "recurring-interval") {
-            let array = this.start_time.split(":");
-            let hour = parseInt(array[0]);
-            let minute = parseInt(array[1]);
-            this.cron = minute + " " + hour + " */" + this.interval_day + " * *";
+            this.cron = "* * * * *"; // Because it is interval, it will be calculated in the run function
```

Comment:
nit: think this can be bit rephrased, feel free to reject

```suggestion
            // For intervals, the pattern is calculated in the run function as the interval-option is set
            this.cron = "* * * * *";
```